interface/globals.php

```diff
@@ -234,6 +235,9 @@
 $twigEnv->addFilter(new Twig_SimpleFilter('translate', function ($string) {
     return xl($string);
 }));
+$twigEnv->addFunction(new Twig_SimpleFunction('setupHeader', function ($asset = null) {
+    Header::setupHeader($asset);
+}));
 
 /** Twig_Loader */
 $GLOBALS['twigLoader'] = $twigLoader;
```

Comment:
Yes, the loader needs to be in the globals array because some controllers/models may need to use a path not in the `templates` dir and you have to add that path to the loader before you render